LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-GC.https.html

```diff
@@ -21,8 +21,11 @@
   ctx.fillStyle = "blue";
   const drawCanvas = () => {
     ctx.fillRect(0, 0, canvas.width, canvas.height);
+    setTimeout(drawCanvas, 50);
```

Comment:
it seems to me that we could avoid all those timers by simply first waiting for the video readyState to go into HAVE_CURRENT_DATA

Reply:
This timeout is simply there to trigger a new video frame every 50 ms. Since we are using webrtc, a frame can be skipped or changed resolution due to bandwidth...